cumulusci/tasks/metadata/package.py

```diff
@@ -46,13 +46,14 @@ class MetadataParserMissingError(Exception):
 class PackageXmlGenerator(object):
     def __init__(
         self,
-        directory,
         api_version,
+        directory=None,
```

Comment:
I'm nervous about changing the call signature because I'm not 100% sure someone else isn't using this, and call sites that are not updated would lead to non-obvious errors. Instead, let's just make the new calls be explicit and pass "." as the directory.